### DISCUSSION TITLE
Is there any file i can change to edit the look of the cursor?

### DISCUSSION THREAD

**Discussion by BusterLundgaard** [flxzt/rnote#767] [Q&A]
I am currently using RNote to record mathematics screencasts. For this, I need to the cursor to be very clear and easy to see for the viewers, so that I can use it as a kind of "pointing stick". Is there any .png or .svg file I can change to edit the look of the pointer, or is it hardcoded in so that i would need to rebuild RNote from scratch? 

Kind regards.

**Comment by flxzt** [comment]
You can change the cursor in the app. Check out the settings:

![Bildschirmfoto vom 2023-08-14 12-11-23](https://github.com/flxzt/rnote/assets/19841886/45ff676a-a33a-4713-a0de-3f6b5450a2d8)

**Reply by Bonk357b** [reply]
Can i somehow hide the pen cursor even when not drawing? Like when hovering over the screen.
